mysql-test/include/percona_bug4977.inc

```diff
@@ -0,0 +1,15 @@
+# PS-4977 : [PS8QA] Flushing the table with read lock could lead to a server
```

Comment:
I don't think the new testcases are required, because this fixes an existing failure: main.mdl_tablespace, please confirm